src/core/__mocks__/agent/multiSigMock.ts

```diff
@@ -0,0 +1,626 @@
+              dt: "2024-08-09T10:26:43.853000+00:00",
+              LEI: "5493001KJTIIGC8Y1R17",
+            },
+          },
+          iss: {
+            v: "KERI10JSON0000ed_",
+            t: "iss",
+            d: "EN3heuFbxVO29fXp87-tisLFKJ7hX2U2gISCRFBLzE0E",
+            i: "EK0IO8nKiSEi6hQKRj5f4YjLkMSSonoxkFLwP1YgycJ1",
+            s: "0",
+            ri: "EGtWoXT4ahu5YnmGCAV8stolM-fgdqCu_eggQ46uSSoi",
+            dt: "2024-08-09T10:26:43.853000+00:00",
+          },
+          anc: {
+            v: "KERI10JSON00013a_",
+            t: "ixn",
+            d: "EOpvfZPwtHf05Di-pGjcj2tyKjFwXAHBJfKuHx4eOsqX",
+            i: "EOfnSCpCa3XmSTUZ7vEgEJoYbruyiYIbl5DYsAwooXTY",
+            s: "4",
+            p: "EHiOcu5i_MGv8EHHSpdRHBsKNtkLc0oTIeXsevRt_kZh",
+            a: [
+              {
+                i: "EK0IO8nKiSEi6hQKRj5f4YjLkMSSonoxkFLwP1YgycJ1",
+                s: "0",
+                d: "EN3heuFbxVO29fXp87-tisLFKJ7hX2U2gISCRFBLzE0E",
+              },
+            ],
+          },
+          d: "EAws1bbmwAdKbviwBqoUZGjI4YB7TyULXSMuO_2otwe-",
+        },
+      },
+      d: "EIwwfdduAceqZpjPogZPFBC5pgdKfRgwsSGrr11aE13A",
+    },
+  },
+  size: 607,
+};
+
+const mockSigsMes = [
+  "ABB5MNMwQNwDxo4hhNYdF2hjzJQfZe6nfjdO_WfjvnxJwkRMrkrbapdjW8BStprjhxASVKWHmWOEFVpGWwmDpyIP",
+];
+
+const mockDtime =
+  "-LA35AACAA-e-exn-FABEPIKswKD9AiVxIqU4QLn14qpNuiAfgVGzoK-HVU0znjC0AAAAAAAAAAAAAAAAAAAAAAAEPIKswKD9AiVxIqU4QLn14qpNuiAfgVGzoK-HVU0znjC-AABAABTAefC5IBObzL5ZteOa6me6iLQXV1v1rTcsBOrJDfk6uwRfR1nxm2DKWxehRMHEdq6YlqxysCdWfVBIvd4t3gH";
+
+const mockResolvedValue = {
+  name: "oobi.ABi0yV6LhFudjaGT4wkCNpVChyHOSFii_idAIAEhyfot",
+  metadata: {
+    oobi: "http://keria:3902/oobi/EH_rgokxkQE886aZf7ZRBgqN2y6aALPAmUvI5haK4yr7/agent/EFv5qAoixFv0WwsekWSqS5ypXQUl2f_N4j8GGURs3QUT",
+  },
+  done: true,
+  error: null,
+  response: {
+    vn: [1, 0],
+    i: "EH_rgokxkQE886aZf7ZRBgqN2y6aALPAmUvI5haK4yr7",
+    s: "0",
+    p: "",
+    d: "EH_rgokxkQE886aZf7ZRBgqN2y6aALPAmUvI5haK4yr7",
+    f: "0",
+    dt: "2024-08-09T11:04:09.343208+00:00",
+    et: "icp",
+    kt: "1",
+    k: ["DLt2-jru3tSgE9Ui2dSRCYy5BJjWsyGXsX7klF_ecylO"],
+    nt: "1",
+    n: ["EIxuzxaSiS6HHdff1M8vzkRlJSFK4YwpD813PPmnBZSe"],
+    bt: "3",
+    b: [
+      "BBilc4-L3tFUnfM_wJr4S4OJanAv_VmF_dJNN6vkf2Ha",
+      "BLskRTInXnMxWaGqcpSyMgo0nYbalW99cGZESrz3zapM",
+      "BIKKuvBwpmDVA4Ds-EpL5bt9OqPzWPja2LigFYZN2YfX",
+    ],
+    c: [],
+    ee: {
+      s: "0",
+      d: "EH_rgokxkQE886aZf7ZRBgqN2y6aALPAmUvI5haK4yr7",
+      br: [],
+      ba: [],
+    },
+    di: "",
+  },
+};
+
+const mockConnectionShortDetail = {
+  id: "ENsj-3icUgAutHtrUHYnUPnP8RiafT5tOdVIZarFHuyP",
+  label: "f4732f8a-1967-454a-8865-2bbf2377c26e",
+  oobi: "http://127.0.0.1:3902/oobi/ENsj-3icUgAutHtrUHYnUPnP8RiafT5tOdVIZarFHuyP/agent/EF_dfLFGvUh9kMsV2LIJQtrkuXWG_-wxWzC_XjCWjlkQ",
+  status: ConnectionStatus.CONFIRMED,
+  connectionDate: new Date().toISOString(),
+};
+
+const mockNotification = {
```

Comment:
This `mockNotification` is used in several tests but it's for an IPEX grant. Why is it used for multisig rotations etc?

Reply:
@Sotatek-BaoHoanga This is still called `mockNotification` and now you added `mockRotation` which doesn't even have a route. Does the route not matter in either case?